src/staff/components/StaffProperties/StaffProperties.tsx

```diff
@@ -147,20 +65,66 @@ const StaffProperties: React.FC<StaffPropertiesProps> = props => {
         </DashboardCard.Title>
       </DashboardCard.Header>
       <DashboardCard.Content>
-        <div className={classes.root}>
+        <Box display="grid" gap={6} __gridTemplateColumns="120px 1fr">
           <div>
-            <div className={classes.avatar}>
+            <Box
+              alignItems="center"
+              borderRadius="100%"
+              display="grid"
+              justifyContent="center"
+              overflow="hidden"
+              position="relative"
+              __height="120px"
+              __width="120px"
+              className="staff-avatar"
+            >
               {hasAvatar ? (
-                <img className={classes.avatarImage} src={staffMember.avatar.url} />
+                <Box as="img" pointerEvents="none" width="100%" src={staffMember.avatar.url} />
               ) : (
-                <div className={classes.avatarDefault}>
-                  <Text>{getUserInitials(data)}</Text>
-                </div>
+                <Box
+                  backgroundColor="accent1"
+                  __height="120px"
+                  __width="120px"
+                  display="flex"
+                  justifyContent="center"
+                >
+                  <Text
+                    color="buttonDefaultPrimary"
+                    fontWeight="bold"
+                    textAlign="center"
+                    __fontSize={35}
+                    __lineHeight="120px"
+                  >
+                    {getUserInitials(data)}
+                  </Text>
+                </Box>
               )}
               {canEditAvatar && (
-                <div className={classes.avatarHover}>
+                <Box
+                  className="avatar-hover"
+                  borderRadius="100%"
+                  opacity="0"
```

Comment:
Try something like this, it works for me and you can get rid of class and this styles:
```
.staff-avatar:hover .avatar-hover {
  opacity: 1;
}
```

```suggestion
                  borderRadius="100%"
                  opacity={{
                      hover: "1",
                      default: "0",
                  }}
```

Reply:
right🤦 makes sense, thanks